### PR TITLE
Freerdp3

### PR DESCRIPTION
**Pull request by joebonrichie**
**Summary**

- Updates freerdp to v3
- Creates a separate freerdp2 package
- Necessary as gnome-remote-desktop 46.0 requires freerdp >= 3.1.0

**Test Plan**

- Connect to remmina over gnome-remote-desktop 46.

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

File: packages/f/freerdp2/package.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+name       : freerdp2
+version    : 2.11.4
+release    : 34
```
Comment (ReillyBrogan) [inline packages/f/freerdp2/package.yml:3]: relnum should be 1 for a new package.

**Review by ReillyBrogan — CHANGES_REQUESTED** [review]
Also builddep/rundep ordering issues.

**Comment by joebonrichie** [comment]
builddep ordering is just due to temporary lack of pkgconfig for freerdp2

**Review by ReillyBrogan — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**